src/media/js/views/langpacks.js

```diff
@@ -0,0 +1,40 @@
+define('views/langpacks',
```

Comment:
good idea to "docstring" our modules explaining its purpose, and a bug number for history